src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -211,6 +213,8 @@ def _get_azure_cli_properties(self):
         set_custom_properties(result, 'PollEndTime', str(self.poll_end_time))
         set_custom_properties(result, 'CloudName', _get_cloud_name())
         set_custom_properties(result, 'ShowSurveyMessage', str(self.show_survey_message))
+        set_custom_properties(result, 'RegionInput', self.region_input)
+        set_custom_properties(result, 'RegionIdentified', self.region_identified)
```

Comment:
If it's only used by `vm create`, I will suggest to add an extension event for vm commands instead of inserting new properties for all